patches/gtsam.patch

```diff
@@ -1,13 +1,121 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8c4229e..9534be1 100644
 --- ./CMakeLists.txt
 +++ ./CMakeLists.txt
-@@ -207,7 +207,7 @@ endif()
+@@ -53,7 +53,7 @@ endif()
+ 
+ # Configurable Options
+ if(GTSAM_UNSTABLE_AVAILABLE)
+-    option(GTSAM_BUILD_UNSTABLE              "Enable/Disable libgtsam_unstable"          ON)
```

Comment:
For the benefit of reducing the size of the patch (and the headaches when the library gets updated), it would be better to leave these the way they are and set the defines in the autobuild file IMO.